### PR TITLE
log 4xx and 5xx errors

### PR DESCRIPTION
**Pull request by lrlna**
i am thinking something like this as a fix for #49

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -80,6 +80,12 @@ Merry.prototype.router = function (opts, routes) {
           var statusCode = err.output.statusCode ||
             (res.statusCode >= 400 ? res.statusCode : 500)
 
+          if (statusCode === 400) {
```
Comment (yoshuawuyts) [inline index.js]: this won't work if we hit something like a `401`

**Comment by lrlna** [comment]
oh lol bahhhh ofc *facepalm*

File: index.js [hunk 2]
```diff
@@ -80,6 +80,12 @@ Merry.prototype.router = function (opts, routes) {
           var statusCode = err.output.statusCode ||
             (res.statusCode >= 400 ? res.statusCode : 500)
 
+          if (statusCode >= 400) {
```
Comment (yoshuawuyts) [inline index.js]: nope, if it's `>= 500` it'll also be `>= 400` - the second condition will never resolve

Comment (lrlna) [inline index.js]: yeppp you're right; will fix + add tests

**Comment by yoshuawuyts** [comment]
Did you cut a release for this too btw?

**Comment by lrlna** [comment]
yeppppp it's the [4.1.4](
https://github.com/yoshuawuyts/merry/commit/6826070ffb0ca6d5321202e63c5c658eaa7d9260)
one ✨

**Comment by yoshuawuyts** [comment]
Swooooosh

**Comment by lrlna** [comment]
🏄🌊🌊
🌊🌊🌊

**Pull request merged**